src/helpers.ts

```diff
@@ -1,14 +1,19 @@
 import { SubscriptionClient } from './client';
-import { print } from 'graphql/language/printer';
+
+/**
+ * @deprecated This method will become deprecated in the new package graphql-transport-ws.
+ * Start using the GraphQLTransportWSClient to make queries, mutations and subscriptions over websockets.
+ */
 
 // Quick way to add the subscribe and unsubscribe functions to the network interface
+// We will move this into a new package in the future
```

Comment:
In the new package we shouldn't have this, but we can keep it in the old one and just print the deprecation warning.